Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml

```diff
@@ -0,0 +1,786 @@
+id: XCloud Cryptomining
+version: -1
+name: XCloud Cryptomining
+description: "Investigates a Cortex XDR incident containing Cloud Cryptomining related alert. \nThe playbook supports AWS, Azure, and GCP and executes the following:\n\n- Cloud enrichment:\n\n    -Collects info about the involved resources\n\n    -Collects info about the involved identities\n\n    -Collects info about the involved IPs\n\n\n- Verdict decision tree\n\n\n- Verdict handling:\n\n -Handle False Positives\n\n -Handle True Positives\n\n -Cloud Response - Generic sub-playbook.\n\n- Notifies the SOC if a malicious verdict was found"
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: c1c09dc8-6340-4782-80fb-ceaab5e45309
+    type: start
+    task:
+      id: c1c09dc8-6340-4782-80fb-ceaab5e45309
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "39"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 720,
+          "y": -200
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: e7062263-5ddf-470e-8a2b-b01082d6513c
+    type: title
+    task:
+      id: e7062263-5ddf-470e-8a2b-b01082d6513c
+      version: -1
+      name: Set verdict
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "48"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 720,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "27":
+    id: "27"
+    taskid: 0e878c77-92fd-4ca9-81f9-56d977cb10d6
+    type: regular
+    task:
+      id: 0e878c77-92fd-4ca9-81f9-56d977cb10d6
+      version: -1
+      name: Set the incident severity
+      description: Optionally increases the incident severity to the new value if it is greater than the existing severity.
+      scriptName: IncreaseIncidentSeverity
```

Comment:
Have you checked how this automation runs on XSIAM? will it raise the alert or the incident? (Cause I dont see any input for the incident)

Reply:
Updates the alert's severity. Of course, if the updated severity has been raised (M->H) for example, the incident's severity will be updated as well.